src/features/home/components/BusinessModule.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { Dimensions, PixelRatio } from 'react-native'
+import styled from 'styled-components/native'
+
+import { BusinessModuleIcon } from 'features/home/assets/BusinessModuleIcon'
+import { NextArrowIcon } from 'features/home/assets/NextArrowIcon'
+import { BusinessPane } from 'features/home/contentful'
+import { Typo, ColorsEnum, getSpacing } from 'ui/theme'
+
+export const BusinessModule = ({ firstLine, secondLine, image, url }: BusinessPane) => {
+  const firstLineWithEndSpace = `${firstLine} `
+  return (
+    <Row>
+      {/** Explicitly defining Margin component for easier use for other components, with gutters */}
+      <Margin />
+      <TouchableContainer>
+        <TouchableHighlight
+          onPress={() => console.log(`Opening ${url}`)} // eslint-disable-line no-console
+        >
+          <ImageContainer>
+            <ImageBackground source={{ uri: image }} testID="imageBusiness">
+              <Container>
+                <BusinessModuleIcon />
+                <StyledText numberOfLines={2}>
+                  <Typo.ButtonText color={ColorsEnum.WHITE}>
+                    {firstLineWithEndSpace}
+                  </Typo.ButtonText>
+                  <Typo.Body color={ColorsEnum.WHITE}>{secondLine}</Typo.Body>
+                </StyledText>
+                <NextArrowIcon />
+              </Container>
+            </ImageBackground>
+          </ImageContainer>
+        </TouchableHighlight>
+      </TouchableContainer>
+      <Margin />
+    </Row>
+  )
+}
+
+// TODO(agarcia): place these constants in a file for style
+const MARGIN_DP = 24
+const BORDER_RADIUS = 8
+
+const imageWidth = Dimensions.get('window').width - 2 * MARGIN_DP
+const imageHeight = PixelRatio.roundToNearestPixel((imageWidth * 116) / 553)
```

Comment:
```suggestion
const imageHeight = PixelRatio.roundToNearestPixel((imageWidth * 116) / 332)
```

![image](https://user-images.githubusercontent.com/10118284/97896436-5915b880-1d35-11eb-894f-e34d68cc69c6.png)